frontend/src/components/beta/common/UtcLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { styled } from "../../../libraries/styles";
+
+const UTC = styled("div", {
+  fontFamily: "SF Mono",
+  fontWeight: 600,
+  fontSize: 12,
+  lineHeight: "20px",
+  color: "#fff",
+  paddingHorizontal: 7,
```

Comment:
I would discuss those odd numbers with @reefoh, I believe they should be dividable by 2 or even by 4.